CHANGELOG.md

```diff
@@ -14,10 +14,12 @@
     - Pulled out the `UiStateManager` from the `BuildConfigEditor` into this assembly and made it public.
 - Exceptions thrown in user-code callbacks no longer cause other callbacks scheduled for that frame to not fire. Instead, the exceptions are caught and logged with Debug.LogException.
 - Upgraded the Worker SDK version to `13.7.1`.
+- The Development Authentication Token will by default be loaded searching for a `DevAuthToken.txt` asset in any of your Resources folder.
```

Comment:
this could probably be clearer to be something along the lines of: 
```suggestion
- Updated the default method of loading a Development Authentication Token to search for a `DevAuthToken.txt` asset at the root of any `Resources` folder.
```